src/app/children/attendance/attendance-month.ts

```diff
@@ -36,15 +37,22 @@ export class AttendanceMonth extends Entity {
   }
 
 
-  student: string; // id of Child entity
-  remarks = '';
-  institution: string;
+  @DatabaseField() student: string; // id of Child entity
+  @DatabaseField() remarks: string = '';
+  @DatabaseField() institution: string;
+
 
   private p_month: Date;
+  @DatabaseField({dataType: 'month'})
```

Comment:
Why do you set the getter as the `DatabaseField`?

Reply:
If the decorator is on the private p_month, the field name saved to the database would be p_month rather than month, too. Having the name "month" seems much cleaner

Reply:
That makes sense but does this work when loading the object from the database to? Will the database field month be mapped to the p_month?

Reply:
I added another unit test in 1fc1923 that proves it is working. As the getter and setter run on the same property name, it's not an issue - and I wanted the loading to use the setter. I agree however, that it makes the code somewhat more confusing